sshfs.rst

```diff
@@ -228,6 +230,16 @@ removed it, but before SSHFS had the time to rename the old file. In
 this case, the rename will still fail.
 
 
+Permission denied when moving files across remote filesystems
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+SSH servers return a generic error when failing to rename across filesystem
```

Comment:
Please add "some" (unless this is actually standard behavior, in that case please provide a link to the standard)

Reply:
This comes from the code (none of the 8 defined SSH_FX_* return codes describe EXDEV) and from the web at https://winscp.net/eng/docs/sftp_codes "Most SSH/SFTP servers support only SFTP version 3 that defines only codes 0 to 8." (Note that even amongst the "extended" error codes none describe EXDEV).  So I changed it to "Most SSH servers".